### PR TITLE
added and fixed partner images

### PR DESCRIPTION
**Pull request by nexussays**

### CODE REVIEW

File: site/src/data/PartnerList.ts [hunk 1]
```diff
@@ -318,6 +318,13 @@ const data: Partner[] = [
       logo: "national-coalition-for-accessible-voting.png",
       dark: true,
    },
+   {
+      id: "nacdd",
+      name: "National Association of Councils on Developmental Disabilities",
+      logo: "nacdd.png",
+      source: "NACDD",
+      dark: true,
```
Comment (nexussays) [inline site/src/data/PartnerList.ts]: @mojowen that white rounded box bkgd was a nice addition btw

File: site/src/components/page-form/page-form.tsx [hunk 2]
```diff
@@ -22,28 +26,49 @@ export class PageForm {
...
+      // see if this is a partner link, e.g., https://powerthepolls.org/aflcio
       const paths = document.location.pathname.split( "/" ).filter( x => x !== "" );
-      const urlParam = paths.length > 0 ? paths[0] : "";
-      // if the field in the URL matches a partner, get their partner ID (source or ID) used, else fallback to querystring arg "source"
-      const partner = ( PartnerList.filter( x => x.id === urlParam.toLowerCase() ) || [null] )[0];
-      const partnerId = partner?.source || partner?.id || getParams()?.source;
+      const urlParam = paths.length > 0 ? paths[0].toLowerCase() : "";
+      const partner = urlParam != null
+         ? ( PartnerList.filter(
+            p => ( p.vanityUrl && p.vanityUrl.toLowerCase() === urlParam ) || p.partnerId.toLowerCase() === urlParam,
```
Comment (mojowen) [inline site/src/components/page-form/page-form.tsx:43]: Thanks for updating this - I realized I just downcased one side

File: site/src/components/page-partners/page-partners.tsx [hunk 3]
```diff
@@ -58,7 +60,7 @@ export class PagePartners {
          <h3-bar>Partners</h3-bar>
          <div class="partner-logos">
             {partners.map( partner => (
-               !partner.founding && partner.logo && ( <div class={partner.dark ? "dark" : ""}><img
+               !partner.founding && partner.logo && ( <div class={partner.logoIsDark ? "dark" : ""}><img
```
Comment (mojowen) [inline site/src/components/page-partners/page-partners.tsx:63]: 👍 better variable

**Review by mojowen — APPROVED** [review]

**Comment by nexussays** [comment]
@mojowen added the `PtpLink` thing, probably should have put that in another branch. But lmk if it's good then this can go.

**Review by mojowen — APPROVED** [review]
Great idea with `PtpLink` - changes look good on my end

**Pull request merged**